solutions/devicesimulation-nohub/single-vm/setup.sh

```diff
@@ -185,6 +186,7 @@ echo ""
 echo "# Format: true | false"                                                                            >> ${ENVVARS}
 echo "# empty => Auth required"                                                                          >> ${ENVVARS}
 echo "export PCS_AUTH_REQUIRED=\"\""                                                                     >> ${ENVVARS}
+echo "export PCS_AUTH_WEBSERVICE_URL=\"\""                                                               >> ${ENVVARS}
```

Comment:
why leave this empty?

Reply:
This is required by config service but are not used by DS solution so passing empty vars